tests/functional/test_config/ppo/hopper_ft.yaml

```diff
@@ -0,0 +1,29 @@
+# @package _global_
```

Comment:
Those are the functional test config to run a "minimal" experiment. Let me know if you have issues with some parameters or want different configurations.